custom/icds_reports/static/js/directives/navigation/navigation.directive.js

```diff
@@ -58,12 +60,25 @@ function NavigationController($window, $rootScope, $scope, $route, $routeParams,
             document.getElementById('nav-menu').style.left = '-300px';
         }
     };
+
+    $scope.reportAnIssue = function () {
+        $scope.closeMenu();
+        if (reportAnIssueUrl) {
+            $window.location.href = reportAnIssueUrl;
```

Comment:
why do we change the window location here?

Reply:
I have no clue. I just copied it from here: 
https://github.com/dimagi/commcare-hq/blob/56293e999790416fb819be9a437449294fe60766/custom/icds_reports/static/js/icds_app.js#L36

Reply:
alright, let's leave it out if we don't know and it works without it. maybe we can remove it there too

Reply:
I'm just anxious that it might create an issue that we could not see now. Can we leave it as is?

Reply:
since it's behind a flag we have the opportunity to do testing before it gets released. i'd rather have less code that works than more code that we are scared to modify because we don't understand it.

Reply:
Ok. You want me to remove it only in mobile dashboard right?